pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -606,6 +607,10 @@ func (br *OVSBridge) GetOFPort(ifName string) (int32, Error) {
 		return 0, NewTransactionError(fmt.Errorf("interface %s not found", ifName), false)
 	}
 	ofport := int32(res[1].Rows[0].(map[string]interface{})["ofport"].(float64))
+	// ofport value -1 means that the interface could not be created due to an error.
```

Comment:
So, -1 must be an error?
What will be returned if the ofport is not assigned by switchd yet?

Reply:
we get a timeout error

Reply:
Right. I now remember we should get a timeout (the code was my me ...)